packages/renderer/src/lib/volume/VolumeColumnActions.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+import VolumeActions from './VolumeActions.svelte';
+import type { VolumeInfoUI } from './VolumeInfoUI';
+
+export let object: VolumeInfoUI;
```

Comment:
Not sure object is well choosen

Reply:
I would rather not change it atm since it spans at least 3 PRs ;-), but if you can think of a better generic property name I'd be glad to PR it early next week.